layouts/partials/tw/nav.html

```diff
@@ -1,137 +1,268 @@
+          :class="toggle ? 'w-5 -translate-y-1.5 -rotate-45' : ''"
+        ></span>
+      </div>
+    </div>
+  </button>
+  <ul
+    class="
+      -top-2 right-1 z-10 ml-1 box-content flex-wrap gap-2 bg-white shadow sm:relative sm:top-auto sm:flex sm:h-auto sm:w-auto sm:max-w-4xl sm:flex-1 sm:flex-nowrap sm:items-center sm:justify-evenly sm:rounded-none sm:border-none sm:bg-transparent sm:bg-none sm:px-16 sm:shadow-none sm:before:content-[] md:overflow-visible md:px-[6.75rem]
+    "
+    :class="toggle ? 'max-sm:absolute max-sm:border rounded border-s-5 flex w-auto h-auto px-4 pt-14 pb-10 sm:pt-0 sm:pb-0 sm:overflow-visible' : 'overflow-hidden w-0'"
+    x-on="toggle"
+  >
+    {{ range site.Menus.main }}
+      <li
+        class="flex h-11 flex-[1_0_45%] items-center text-sm leading-none sm:flex-[0_0_auto] md:justify-center"
+      >
         <a
-          href="/search/"
-          class="site-nav:item"
-          data-ga-category="search"
-          :data-ga-action="'modal:search:open'"
-          x-data
-          @click.prevent="$dispatch('modal:search:open')"
+          class="flex-[0_0_auto] font-bold uppercase hover:underline hover:underline-offset-2 sm:font-normal"
+          href="{{ .URL }}"
+          data-ga-category="item"
+          >{{ .Name }}</a
         >
-          <svg class="site-nav:svg-icon" aria-label="Search">
-            {{ partial "helper/svg.html" "search" }}
-          </svg>
-        </a>
+      </li>
+    {{ end }}
+    {{ range site.Menus.main_about }}
+      <li
+        class="
+          mt-4 flex-[0_0_100%] flex-wrap items-center text-sm leading-none sm:absolute sm:z-10 md:my-0 md:flex md:flex-[0_0_auto] md:overflow-visible
+        "
+        :class="toggle ? 'sm:block sm:-right-16 sm:-top-6 sm:pt-2 sm:pb-3 sm:pr-8 sm:pl-4 sm:rounded-tr sm:bg-white sm:overflow-visible sm:w-48 sm:h-auto md:absolute md:h-auto md:-top-4 md:pt-[15px] box-content sm:border-s-5 sm:border-r sm:border-t' : 'sm:overflow-hidden sm:w-0 sm:h-11 md:top-0 md:right-1'"
+      >
         <a
-          class="site-nav:donate-button"
-          href="/donate/"
-          data-ga-category="donate"
-        >
-          <svg
-            class="site-nav:svg-icon"
-            viewBox="0 0 20 20"
-            aria-hidden="true"
-            style="top: 1px; margin-right: 2px"
-          >
-            {{ partial "helper/svg.html" "heart" }}
-          </svg>
-          Support Us
-        </a>
-      </div>
-      <div class="site-nav:mobile">
-        <div
-          x-data="{ isOpen: false }"
-          x-on:keydown.escape.prevent.stop="isOpen = false; $refs.button.focus()"
-          x-id="['dropdown-button']"
-          style="position: relative;"
+          class="
+            sm:h block w-full flex-1 border-b py-4 font-bold uppercase hover:underline hover:underline-offset-2 sm:flex sm:w-full sm:items-center sm:capitalize
+          "
+          :class="toggle ? 'sm:py-0 sm:h-11 md:justify-start' : 'md:py-0 md:justify-end'"
```

Comment:
I think it would be simpler to just duplicate the top level items inside and out of the pop up menu than to have the inner items changing classes based on the toggle.

Reply:
I guess the tricky part of just having a second set of menu items is that you want the button animation, and that requires the button to be visible before and after. I think you could do it though if the toggle just transforms the container of the button and shows/hides a second set of items that are next to the button. Those could be x-cloaked and hidden from first view to fix the flicker. Definitely a tricky one though.

Reply:
I think this works better from a markup structure and maintenance perspective as a single ul inside the nav. This way we can keep the HTML as structure with CSS adding presentation and JS for DOM manipulation